config/sync/user.role.site_manager.yml

```diff
@@ -7,23 +7,74 @@ label: 'Site Manager'
 weight: -3
 is_admin: null
 permissions:
```

Comment:
- We'll need to add the site title and some things to the config pages entity.
- can't edit metadata at the moment